controllers/v1/cloud-services/files.js

```diff
@@ -0,0 +1,225 @@
+   * @method
+   * @name preSignedUrls
+   * @param  {Request}  req  request body.
+   * @param  {Array}  req.body.fileNames - list of file names
+   * @param  {String}  req.body.bucket - name of the bucket
+   * @returns {JSON} Response with status and message.
+   */
+
+  async preSignedUrls(req) {
+    return new Promise(async (resolve, reject) => {
+      try {
+        let signedUrl = await filesHelpers.preSignedUrls(
+          req.body.request,
+          req.userDetails.userInformation.userId
+        );
+        signedUrl["result"] = signedUrl["data"];
+        return resolve(signedUrl);
+      } catch (error) {
+        return reject({
+          status:
+            error.status || HTTP_STATUS_CODE.internal_server_error.status,
+
+          message:
+            error.message || HTTP_STATUS_CODE.internal_server_error.message,
+
+          errorObject: error,
+        });
+      }
+    });
+  }
+
+  /**
+   * @api {post} /project/v1/cloud-services/files/getDownloadableUrl
+   * Get downloadable URL.
+   * @apiVersion 1.0.0
+   * @apiGroup Gcp
+   * @apiHeader {String} X-authenticated-user-token Authenticity token
+   * @apiParamExample {json} Request:
+   * {
+   *     "filePaths": ["5e1c28a050452374e1cf9841/e97b5582-471c-4649-8401-3cc4249359bb/cdv_photo_117.jpg"]
+   * }
+   * @apiSampleRequest /project/v1/cloud-services/files/getDownloadableUrl
+   * @apiUse successBody
+   * @apiUse errorBody
+   * @apiParamExample {json} Response:
+   * {
+   *  "status": 200,
+   *  "message": "Url's generated successfully",
+   *  "result": [{
+   *  "filePath": "5e1c28a050452374e1cf9841/e97b5582-471c-4649-8401-3cc4249359bb/cdv_photo_117.jpg",
+   *  "url": "https://storage.googleapis.com/download/storage/v1/b/sl-dev-storage/o/5e1c28a050452374e1cf9841%2Fe97b5582-471c-4649-8401-3cc4249359bb%2Fcdv_photo_117.jpg?generation=1579240054787924&alt=media"
+   * }]
+   */
+
+  /**
+   * Get Downloadable URL from cloud service.
+   * @method
+   * @name getDownloadableUrl
+   * @param  {Request}  req  request body.
+   * @returns {JSON} Response with status and message.
+   */
+
+  async getDownloadableUrl(req) {
+    return new Promise(async (resolve, reject) => {
+      try {
+        let downloadableUrl = await filesHelpers.getDownloadableUrl(
+          req.body.filePaths
+        );
+
+        return resolve(downloadableUrl);
+      } catch (error) {
+        return reject({
+          status:
+            error.status || HTTP_STATUS_CODE.internal_server_error.status,
+
+          message:
+            error.message || HTTP_STATUS_CODE.internal_server_error.message,
+
+          errorObject: error,
+        });
+      }
+    });
+  }
+
+  /**
+      * upload the file to the cloud .
+      * @method
+      * @name upload
+      * @param  {Request}  req  request body.
+      * @returns {JSON} Response with status .
+      * @apiParamExample {json} Response:
+     * {
+            "status": 200
+       }
+     */
+
+  async upload(req) {
```

Comment:
@Prajwal17Tunerlabs this api is tested? Add comments to the code to increase readability

Reply:
this api is removed since we are not using it anymore